packages/next/server/dev/hot-reloader.ts

```diff
@@ -414,6 +420,42 @@ export default class HotReloader {
       )
   }
 
+  private async getVersionInfo(span: Span) {
+    const versionInfoSpan = span.traceChild('get-version-info')
+    return versionInfoSpan.traceAsyncFn<VersionInfo>(async () => {
+      // TODO: Get the correct values
+      const installed = semver.parse('13.0.6')
```

Comment:
Do we have the installed version somewhere, or is the best option to import `package.json` and check the `version`?

Need to figure out how to get the latest/canary versions. The [package metadata](https://registry.npmjs.org/next) is huge (9+ MB!) and significantly slows down bootup.


My idea was to stream the response and stop reading after a few lines. An alternative solution was suggested here: https://github.com/vercel/next.js/pull/37750/files/b1680cb4abecba65b9f89e232edc3ccdadc6e335#r1000786356, but it seems that even unpkg.com can take several seconds sometimes.

Reply:
Will use a cached endpoint.